### PR TITLE
Ch folder structure feedback 165399350

### PR DESCRIPTION
**Pull request by shalu93**
ch: changing folder structures
-changed folder structure
-added validations
-changed basic route structure

[finishes #165399350]

### CODE REVIEW

**Comment by IyiKuyoro** [comment]
Please add the PR message using the template I provided.

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@ node_modules/
 .env
 .nyc_output
 coverage
-travis.rb
\ No newline at end of file
+travis.rb
+.eslintrc.json
```
Comment (IyiKuyoro) [inline .gitignore]: Please remove the .eslint.json from .gitignore

File: UI/HTML/Amend_admin_or_staff_user_acct.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+  function getUrlVars() {
+    var vars = {};
+    var parts = window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
+        vars[key] = value;
+    });
+    return vars;
+}
+   var dynamicContent = getUrlVars()["AccountType"];
+$(document).ready(function() {
```
Comment (IyiKuyoro) [inline UI/HTML/Amend_admin_or_staff_user_acct.html]: I can still see inline javascript here.

Move all javascript into a seperate file.

**Review by IyiKuyoro — CHANGES_REQUESTED** [review]
Good work on the folder structure. Please attend to the two comments raised.

File: UI/HTML/Amend_admin_or_staff_user_acct.html (hunk 2 above)
Comment (shalu93) [reply UI/HTML/Amend_admin_or_staff_user_acct.html]: changed it

**Pull request merged**